bin/node/runtime/src/weights/pallet_balances.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Weights for the Balances Pallet
+
+use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};
```

Comment:
```suggestion
use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};

```
nit: newline between imports and code

Reply:
(would need to adjust the generator, not sure it's worth it or in scope)

Reply:
Happy to fine tune the generator in a follow up. Need to do things like add a comment line:

```
THIS FILE IS AUTOGENERATED
```

and stuff